### PR TITLE
Change project for Release Candidate clusters

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- Change project name in RC job to  `sap-kyma-prow-workloads` (update and regenerate template)
- Configure TTL 
- Set different (admin) permissions for RC clusters for `kyma_developers` group

**Related issue(s)**
[Issue #1750](https://github.com/kyma-project/test-infra/issues/1750)

### CODE REVIEW

File: prow/scripts/cluster-integration/helpers/provision-gke-cluster.sh [hunk 1]
```diff
@@ -56,6 +57,7 @@ if [ "${STACKDRIVER_KUBERNETES}" ];then GCLOUD_PARAMS+=("--enable-stackdriver-ku
 if [ "${CLUSTER_USE_SSD}" ];then GCLOUD_PARAMS+=("--disk-type=pd-ssd"); fi
 #Must be added at the end to override --num-nodes parameter value set earlier.
 if [ "${PROVISION_REGIONAL_CLUSTER}" ] && [ "${CLOUDSDK_COMPUTE_REGION}" ] && [ "${NODES_PER_ZONE}" ];then GCLOUD_PARAMS+=("--region=${CLOUDSDK_COMPUTE_REGION}" "--num-nodes=${NODES_PER_ZONE}"); fi
+if [ "${GCLOUD_SECURITY_GROUP_DOMAIN}" ]; then GCLOUTD_DAPAMS+=("--security-group=gke-security-groups@${GCLOUD_SECURITY_GROUP_DOMAIN}"); fi
```
Comment (kasiakepka) [inline prow/scripts/cluster-integration/helpers/provision-gke-cluster.sh]: GCLOUTD_DAPAMS+ ?

Comment (pPrecel) [reply prow/scripts/cluster-integration/helpers/provision-gke-cluster.sh]: ups

File: prow/scripts/cluster-integration/kyma-gke-release-candidate.sh [hunk 2]
```diff
@@ -44,6 +44,9 @@ fi
 export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
 export KYMA_SOURCES_DIR="${KYMA_PROJECT_DIR}/kyma"
 export TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers"
+export TTL_HOURS=120 #5 days
+export GCLOUD_SECURITY_GROUP="kyma_developers"
+export GCLOUD_SECURITY_GROUP_DOMAIN="sap.com"
```
[the remark this answers is not in the document]
Comment (pPrecel) [reply prow/scripts/cluster-integration/kyma-gke-release-candidate.sh]: Good idea

**Pull request merged**